### PR TITLE
samples: kernel: metairq: Update ReadMe

### PR DESCRIPTION
**Pull request by aasthagr**
Update ReadMe for metairq sample application.

Signed-off-by: Aastha Grover <aastha.grover@intel.com>

### CODE REVIEW

**Review by andyross — DISMISSED** [review]
Looks good.  One note is that I'd be a little hesitant to recommend running this on an emulation platform (e.g. qemu_x86) by default, as (unless CONFIG_QEMU_ICOUNT is used) that allows host scheduling to interrupt the emulator and mess things up in ways that make it look like the metairq thread was "preempted" (which... it was, sorta).

File: samples/kernel/metairq_dispatch/README.rst [hunk 1]
```diff
@@ -63,3 +63,85 @@ preemption behavior, it does not run without modification on
...
+Sample Output
+=============
+
+Sample output shown below is from qemu_x86. The numbers will
+look different based upon the clock frequency of the board and other
+factors.
+
+Note: The 'real' values may be significantly higher than corresponding 'proc' values
+for non-cooperative threads like T2 and T3 which is attributed to delays due
+to thread preemption.
```
[the remark this answers is not in the document]
Comment (aasthagr) [reply samples/kernel/metairq_dispatch/README.rst:90]: @andyross I think I should include the word "intended" here to avoid confusion. Thanks.

**Review by andyross — DISMISSED** [review]
Looks good here.

File: samples/kernel/metairq_dispatch/README.rst [hunk 2]
```diff
@@ -63,3 +63,85 @@ preemption behavior, it does not run without modification on
...
 k_busy_wait()) where they will voluntarily release the CPU.
+
+Building and Running
+********************
+
+This application can be built and executed on frdm_k64f as follows:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/kernel/metairq_dispatch
+   :host-os: unix
```
Comment (MaureenHelm) [inline samples/kernel/metairq_dispatch/README.rst:74]: Remove this line

File: samples/kernel/metairq_dispatch/README.rst [hunk 3]
```diff
@@ -63,3 +63,85 @@ preemption behavior, it does not run without modification on
...
+   :board: frdm_k64f
+   :goals: run
+   :compact:
+
+To build for another board, change "frdm_k64f" above to that board's name.
+
+Sample Output
+=============
+
+Sample output shown below is from qemu_x86. The numbers will
```
Comment (MaureenHelm) [inline samples/kernel/metairq_dispatch/README.rst:84]: It's odd to use `frdm_k64f` in the build command, but show sample output from `qemu_x86`. Please be consistent with the board

**Review by MaureenHelm — CHANGES_REQUESTED** [review]

File: samples/kernel/metairq_dispatch/README.rst (hunk 3 above)
Comment (aasthagr) [reply samples/kernel/metairq_dispatch/README.rst]: Sorry, my bad. It is by mistake that qemu_x86 is mentioned here. The command and output are both for the frdm_k64f board. Fixed this. Thanks!!

File: samples/kernel/metairq_dispatch/README.rst (hunk 2 above)
Comment (aasthagr) [reply samples/kernel/metairq_dispatch/README.rst]: Done.

**Review by andyross — DISMISSED** [review]
Refresh +1

File: samples/kernel/metairq_dispatch/README.rst [hunk 4]
```diff
@@ -63,3 +63,84 @@ preemption behavior, it does not run without modification on
...
+
+Building and Running
+********************
+
+This application can be built and executed on frdm_k64f as follows:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/kernel/metairq_dispatch
+   :board: frdm_k64f
+   :goals: run
```
Comment (MaureenHelm) [inline samples/kernel/metairq_dispatch/README.rst:75]: `run` is used for qemu, but you need to use `flash` for real hardware
```suggestion
   :goals: build flash
```

**Review by MaureenHelm — CHANGES_REQUESTED** [review]
Sorry I didn't catch this in my earlier review.

File: samples/kernel/metairq_dispatch/README.rst (hunk 4 above)
Comment (aasthagr) [reply samples/kernel/metairq_dispatch/README.rst]: Fixed.

**Review by ceolin — APPROVED** [review]

**Review by peter-mitsis — APPROVED** [review]

**Review by MaureenHelm — APPROVED** [review]

**Pull request merged**